src/components/LoginForm.jsx

```diff
@@ -0,0 +1,223 @@
+      onChange: () => {
+        console.log("in onChange func");
+      }
+    },
+    email: {
+      type: "email",
+      id: "email",
+      label: "Email",
+      value: "",
+      placeholder: "Enter your email",
+      classes: {
+        wrapperClass: "k",
+        inputClass: "l",
+        errorClass: "k"
+      },
+      validations: [
+        {
+          type: "email",
+          args: []
+        }
+      ],
+      onBlur: () => {},
+      onFocus: () => {},
+      onChange: () => {
+        console.log("in onChange func");
+      }
+    },
+    vehicle: {
+      id: "vehicle",
+      name: "vehicle",
+      type: "checkbox-group",
+      value: {},
+      classes: {
+        wrapperClass: "k",
+        errorClass: "k"
+      },
+      onChange: e => {
+        console.log({ target: e.target });
+      },
+      options: [
+        {
+          id: "car",
+          label: "Car",
+          value: "car",
+          classes: {
+            wrapperClass: "k",
+            inputClass: "l",
+            errorClass: "k"
+          }
+        },
+        {
+          id: "bike",
+          label: "Bike",
+          value: "bike",
+          classes: {
+            wrapperClass: "k",
+            inputClass: "l",
+            errorClass: "k"
+          }
+        }
+      ]
+    },
+    subscribe: {
+      type: "checkbox",
+      id: "subscribe",
+      label: "Choose to get updates.",
+      name: "subscribe",
+      value: "subscribe",
+      initialChecked: false,
+      classes: {
+        wrapperClass: "k",
+        inputClass: "l",
+        errorClass: "k"
+      },
+      onBlur: () => {},
+      onFocus: () => {},
+      onChange: () => {
+        console.log("in onChange func");
+      }
+    },
+    gender: {
+      id: "gender",
+      label: "Select Gender",
+      type: "radio",
+      name: "gender",
+      value: "",
+      classes: {
+        wrapperClass: "k",
+        errorClass: "k"
+      },
+      onChange: e => {
+        console.log({ target: e.target });
+      },
+      validations: ["req"],
+      options: [
+        {
+          id: "male",
+          label: "Male",
+          value: "male",
+          classes: {
+            wrapperClass: "k",
+            inputClass: "l"
+          }
+        },
+        {
+          id: "female",
+          label: "Female",
+          value: "female",
+          classes: {
+            wrapperClass: "k",
+            inputClass: "l"
+          }
+        }
+      ]
+    },
+    maritalStatus: {
+      type: "select",
+      id: "maritalStatus",
+      label: "Marital Status ",
+      placeholder: "Select Your Gender",
+      value: "",
+      classes: {
+        wrapperClass: "k",
+        inputClass: "l",
+        errorClass: "k"
+      },
+      options: [
+        { name: "Single", value: "single", className: "hek" },
+        { name: "Married", value: "married" }
+      ],
+      onBlur: () => {},
+      onFocus: () => {},
+      onChange: () => {}
```

Comment:
You should always try to provide arguments

Reply:
done